### PR TITLE
Edu 1797 b

### PR DESCRIPTION
**Pull request by fairlydurable**
## What does this PR do?

Updates P0 support time in both locations: support-support-guarantees.md and pricing-support.md

## Notes to reviewers

Ignore and close if needed

### CODE REVIEW

File: websites/docs.temporal.io/package.json [hunk 1]
```diff
@@ -39,10 +39,12 @@
     "@mdx-js/react": "^3.0.0",
     "chart.js": "^4.4.1",
     "clsx": "^1.1.1",
+    "docusaurus": "^1.14.7",
```
Comment (flossypurse) [inline websites/docs.temporal.io/package.json:42]: Curious why these dependencies are being added here?

File: websites/docs.temporal.io/package.json [hunk 2]
```diff
@@ -39,10 +39,12 @@
...
     "path": "^0.12.7",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-player": "^2.6.0",
+    "snipsync": "^1.8.0",
```
Comment (flossypurse) [inline websites/docs.temporal.io/package.json:47]: This shouldn't be added, it is added at the root package.json level already.

File: docs-src/cloud/pricing-support.md [hunk 3]
```diff
@@ -18,10 +18,10 @@ They also investigate your Workflows to optimize their environments and possibly
...
-| Pricing (per month) | Greater of $200 or 10% monthly usage                                                        | Greater of $2,000 or 10% monthly usage                                                 |
+|                     | **Basic**                                                                          | **Premium**                                                                            |
+| ------------------- | ---------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------- |
+| Response times      | P0: 1 hour<br/>P1: 4 business hours<br/>P2: 1 business day<br/>P3: 2 business days | P0: 30 minutes<br/>P1: 1 business hour<br/>P2: 4 business hours<br/>P3: 1 business day |
+| Pricing (per month) | Greater of $200 or 10% monthly usage                                               | Greater of $2,000 or 10% monthly usage                                                 |
```
Comment (flossypurse) [inline docs-src/cloud/pricing-support.md:24]: Any idea why "ter of $2" is highlighted here?

File: websites/docs.temporal.io/package.json (hunk 1 above)
Comment (fairlydurable) [reply websites/docs.temporal.io/package.json:42]: I am not sure. This seems to be generated via the artifact creation process with yarn. There were some hiccups along the way where it wasn't building things right, so I went back to my original method.

File: websites/docs.temporal.io/package.json (hunk 2 above)
Comment (fairlydurable) [reply websites/docs.temporal.io/package.json:47]: Ditto. Sorry about that. I did not add.

**Pull request merged**